src/scripts/install_machine.py

```diff
@@ -54,11 +54,35 @@ def get_graphics_capabilies(connection):
 
 
 def prepare_graphics_params(connection):
+    # Use ipv6 as workaround for https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
+    spice_address = '::1'
+
+    # Check to make sure loopback has an IPv6 address assigned
+    # if not default back to IPv4
+    try:
+        if subprocess.run(['ip'], shell=True).returncode == 0:
+            ipv6_loopback = subprocess.check_output(['ip', '-6', 'addr', 'show', 'dev', 'lo'])
```

Comment:
Please avoid `shell=True` and running this command twice. You can use [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run), it defaults to `check=False` and you can check the error code for success and then use the output. You should also use `text=True` and drop the `str()` below

Reply:
Just got those updated, thanks I'll keep that in mind for future